docs/1-introduction/1-2-debug/1-2-4-renderdoc.md

```diff
@@ -1 +1,51 @@
 # RenderDoc
+
+Typically while developing an application there will be complications that cannot be easily deduced by watching the program execute or by reading the code; in a situation like this one will use a debugger to monitor the state of the program to locate the bug, and this is no different for graphics programming. 
+
+Graphics programmers will use what is called a "graphics debugger" when working with a graphics API to monitor API calls, pipeline state, data within the pipeline, etc. 
+
+While there are multiple graphics debuggers provided by multiple vendors such as Nvidia's Nsight, AMD's Radeon Graphics Profiler (Does not directly support DX11), Intel's Graphics Performance Analyzers, and Microsoft's PIX, we will instead be using a cross-platform open-source option, RenderDoc.
+
+To download RenderDoc, [click here](https://renderdoc.org/) and follow the instructions on the webpage.
+
+![Screenshot](https://cdn.discordapp.com/attachments/885023294586224673/956374536671084586/image4.png)
```

Comment:
We cant assume the public has access to all these discord links :) Therefore i suggest the following (we could discuss which option might make most sense:

Create an `images` folder within the folder of the edited/created markdown file, and store the images there, maybe like

images/x-x-x-whateverchapteritwas-1.png/gif/mp4

while -1 is the first image -2 the 2nd andso forth

or 2nd option, we go without the separate folder and store the images in the same level

to link the images you can use relative paths so `images/x-x-x-chapter-nr.png` or `x-x-x-chapter-nr.png`